templates/partials/answers/mobilenumber.html

```diff
@@ -0,0 +1,22 @@
+{% from "components/input/_macro.njk" import onsInput %}
+
+{% set input = form.fields[answer.id] if input is undefined else input %}
+
+{% set config = {
+  "id": answer.id,
+  "name": input.name,
+  "value": input._value() | e,
+  "label": {
+    "id": answer.id ~ "-label",
+    "text": answer.label,
+    "description": answer.description
+  },
+  "attributes": {
+    "data-qa": "input-text"
+  },
+  "legend": question_title,
+  "legendClasses": "u-vh",
+  "error": error
+} %}
```

Comment:
This should conform the telephone input defined in: https://ons-design-system.netlify.app/components/input/
Such as: Setting the type to tel to bring up the correct keypad on touch devices.